src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -839,6 +979,7 @@ private static void setProperties(ModuleInputConfiguration.Builder inputConfigBu
 
         final String propertyContent = readPropertiesContent(beginLineNo, lines);
         final Map<Object, Object> properties = loadProperties(propertyContent);
+        validateDefaultProperties(properties, getDefaultProperties(inputFilePath, moduleName));
```

Comment:
this way avoid feature envy.
```suggestion
        validateDefaultProperties(properties, getDefaultProperties(getFullyQualifiedClassName(filePath, moduleName)));
```